webapp/views/map/script.js

```diff
@@ -33,21 +33,15 @@ const baseLayers = {
   "OSM Humanitarian style": hot
 }
 
-const groupedOverlays = {
-  "Basemap": {
-    [services.waterways.displayName]: createWms(services.waterways),
-    [services.roads.displayName]: createWms(services.roads),
-    [services.buildings.displayName]: createWms(services.buildings),
-    [services.basemapBbox.displayName]: createWms(services.basemapBbox),
-    [services.selection.displayName]: createWms(services.selection)
-  },
-  "Time map": {
-    [services.fromPoints.displayName]: createWms(services.fromPoints),
-    [services.viaPoints.displayName]: createWms(services.viaPoints),
-    [services.strickenArea.displayName]: createWms(services.strickenArea),
-    [services.timeMap.displayName]: createWms(services.timeMap)
-  }
-};
+// Configure the layer switcher
+const groupedOverlays = {}
+const groups = [...new Set(services.map(ser=>ser.group))]
```

Comment:
How can I control the ordering of the groups?

Reply:
By putting them in order in the config.js.